CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ CHANGELOG
 [Next release](https://github.com/rebing/graphql-laravel/compare/6.1.0...master)
 --------------
 
+2020-xx-xx, 6.2.0-rc1
+---------------------
```

Comment:
Not necessary, just remove the two lines -> on release, this will be added, thanks